src/lib/fetchNearbyPlaces.js

```diff
@@ -0,0 +1,36 @@
+//creates promise to fetch data
+const fetchNearbyPlacesPromise = async (lat, lng) => {
+    let url = `https://segdeha.com/api/nearby.php?lat=${lat}&lng=${lng}`
+    let response = await fetch(url);
+
+    let output;
+    if(response.ok){
+        output = await response.json();
+
+    } else {
+        console.log("Error" + response.status);
+        output = { };
```

Comment:
Wondering if you need to set `output = { query: { pages: [] } };` here to prevent errors in this case?

Reply:
In other words, mirror the structure of what you expect to be returned if the call is successful